lua/entities/npc_lambdaplayer.lua

```diff
@@ -327,6 +326,8 @@ function ENT:SetupDataTables()
     self:NetworkVar( "String", 0, "LambdaName" ) -- Player name
     self:NetworkVar( "String", 1, "WeaponName" )
     self:NetworkVar( "String", 2, "ProfilePicture" )
+    self:NetworkVar( "String", 3, "State" ) -- The state we are in. See sv_states.lua
```

Comment:
This can't work because there can only be 3 networked string vars 0, 1,  and 2. Networking anything else will require NW or NW2

Reply:
Huh, didn't know that it works like that. I guess you can just ignore that commit and push the other ones.